ReleaseHistory.md

```diff
@@ -10,6 +10,7 @@
 * DEP: Update `Microsoft.Data.SqlClient` from 2.1.7 to 5.2.2 so its dependencies `Microsoft.IdentityModel.JsonWebTokens` and `System.IdentityModel.Tokens.Jwt` upgrade to non-vulnerable version 6.35.0 (https://github.com/dotnet/aspnetcore/security/advisories/GHSA-59j7-ghrg-fj52).
 * BUG: Resolve process hangs when a file path is provided with a wildcard, but without a `-r` (recurse) flag during the multi-threaded analysis file enumeration phase.
 * BUG: Fix error `ERR997.NoValidAnalysisTargets` when scanning symbolic link files.
+* BUG: Fix error `ERR997.NoValidAnalysisTargets` when wildcard patterns (e.g., *.txt) for file types are provided to `OrderedFileSpecifier`. Previously, the code did not account for these wildcards. Now, the code has been updated to properly handle these patterns, ensuring accurate directory detection even with file type filters.
 * BUG: Fix `ERR999.UnhandledEngineException: System.IO.FileNotFoundException: Could not find file` when a file name or directory path contains URL-encoded characters.
```

Comment:
AI is ready to author a PR description but not yet ready for our release notes. :)

How's this:

* BUG: Fix error `ERR997.NoValidAnalysisTargets` when passing wildcard patterns (e.g., `*.txt`) to `OrderedFileSpecifier`. A recent change limited our wildcard support strictly to use of `*` only.